### ISSUE TITLE
Dispensable (MyHealth@EU requirement catalogue) in logical model and FHIR

### ISSUE THREAD

**Issue by rlindstrm** [Xt-EHR/xt-ehr-common#33]
For crossborder prescription, there is a dispensable/non-dispensable flag on prescriptions.
Only active prescriptions are sent crossborder, but NCP has a set of rules to choose which one of those are dispensable crossborder and which are not. This decision is different for every country, the rules are for example:
- All narcotic product prescriptions non-dispensable (all countries)
- Medications with too general dose form non-dispensable (Estonia)
- Prescription where product amount is given as a time duration, e.g "1 year supply" (Finland)
- etc.

This data element is not filled in at the time of creating the prescription and it is not relevant on a national level. Should we add it to our logical model or FHIR profile or is it something so specific that we would leave it for MyHealth@EU to add?

**Comment by costateixeira** [comment]
Which NCP chooses which lines are dispensable crossborder? I guess of the country of prescription? Are the rules conveyed in the message? Or just a flag?

**Comment by rlindstrm** [comment]
Country A NCP (not prescription system!) decides according to some set of rules that the prescription is not dispensable crossborder. 

Today, the reasoning for this is not included in the message (which means pharmacists are defenseless from the anger of the patient who knows for a fact that this is a valid prescription).

**Comment by costateixeira** [comment]
so, a big red flag that is appended to the prescription and says "not to be dispensed outside of country of original prescription"?

**Comment by rlindstrm** [comment]
Yes, and it functionally stops you from sending the dispensation. 
It says "not dispensable". It does not say "... outside of country of original" because you only see that flag when you are a pharmacist in another country. 
Of course, we can discuss if these flags should appear to domestic users as well in some contexts. It might be helpful for someone who has the urge to rely on crossborder services.

**Comment by rlindstrm** [comment]
Added to the data model as a separate element 'dispensable'. Description implies it's more relevant for crossborder.

**Comment by rlindstrm** [comment]
Removed from logical model as decided in Task 6.2 meeting 06.12.2024. 
If it is needed by MyHealth@EU in the future FHIR specification, they can add this as an extension, as there is no clean mapping to FHIR anyway.
